layers/+tools/shell/packages.el

```diff
@@ -291,3 +290,23 @@
                             eshell-mode-hook
                             shell-mode-hook
                             term-mode-hook)))
+
+(defun shell/init-vterm ()
+  (use-package vterm
+    :defer t
+    :commands (vterm vterm-other-window)
+    :init
+    (progn
```

Comment:
can't we just use `(make-...-command "vterm" vterm)` ? It seems to work.

Reply:
Maybe it changed, IIRC it did not at the time I first wrote this. I will test again and correct if it works.

Reply:
Tested, it worked, it's fixed :sparkles: